### PR TITLE
Enhancement/pwa

### PR DESCRIPTION
**Pull request by VitaliyLang**
add icon to manifest, fix responsive of the form, change scroll on screen scale

### CODE REVIEW

File: src/js/components/LandingPage/LandingPage.jsx [hunk 1]
```diff
@@ -44,21 +44,20 @@ class LandingPage extends React.Component {
...
+    if (window.innerWidth < 1170 || window.devicePixelRatio != 1) return;
     if (e.type === 'keydown') {
-      if (e.keyCode === 40 && this.state.sections[this.state.currentSection + 1]) {
-        scrollTo(this.state.sections[this.state.currentSection + 1])
-        prevent();
-      } else if (e.keyCode === 38 && this.state.sections[this.state.currentSection - 1]) {
-        scrollTo(this.state.sections[this.state.currentSection - 1])
-        prevent();
-      }
+        if (e.keyCode === 40 && this.state.sections[this.state.currentSection + 1]) {
```
Comment (chapran-leobit-acc) [inline src/js/components/LandingPage/LandingPage.jsx]: Indentation broken (in all changes to the file)

File: src/js/components/lib/ModalWindow.jsx [hunk 2]
```diff
@@ -4,39 +4,41 @@ import ReactDOM from 'react-dom';
...
+            this.props.history.push('/');
+        }
+    }
+    preventScroll(e) {
+        e.preventDefault();
+        e.stopPropagation();
+    }
+    componentDidMount() {
+        appRoot.appendChild(this.el);
+        document.body.classList.add('modal-opened');
```
Comment (chapran-leobit-acc) [inline src/js/components/lib/ModalWindow.jsx]: broken css class (correct was "modal_opened")

File: src/js/components/lib/ModalWindow.jsx [hunk 3]
```diff
@@ -4,39 +4,41 @@ import ReactDOM from 'react-dom';
...
+    }
+    componentWillUnmount() {
+        document.body.classList.remove('modal-opened');
```
Comment (chapran-leobit-acc) [inline src/js/components/lib/ModalWindow.jsx]: same as previous

File: src/js/components/lib/ModalWindow.jsx [hunk 4]
```diff
@@ -4,39 +4,41 @@ import ReactDOM from 'react-dom';
...
-  constructor(props) {
-    super(props);
-    this.el = document.createElement('div');
-    this.closeModal = this.closeModal.bind(this);
-  }
-  closeModal(e) {
-    if (e.target.className.match(/\bmodal_window\b/)) {
-      this.props.history.push('/');
+    constructor(props) {
+        super(props);
```
Comment (chapran-leobit-acc) [inline src/js/components/lib/ModalWindow.jsx]: indentation broken

File: src/pwa/manifest.json [hunk 5]
```diff
@@ -1,21 +1,26 @@
...
-      "type": "../assets/img/svg"
-    }
-  ],
-  "start_url": "./index.html",
-  "display": "standalone",
-  "orientation": "portrait",
-  "background_color": "#ffffff",
-  "theme_color": "#ffffff"
-}
+    "short_name": "Leobit",
```
Comment (chapran-leobit-acc) [inline src/pwa/manifest.json]: indentation incorrect (?!)

File: src/scss/components/_DefaultForm.scss [hunk 6]
```diff
@@ -35,14 +35,17 @@
...
+        display: flex;
+        align-items: center;
         label {
             display: inline-block;
             width: auto;
             border-radius: 20px;
             color: $leobit_theme_color;
             border: 1px solid currentColor;
-            padding: 10px 15px;
+            padding: 1px 15px;
```
Comment (chapran-leobit-acc) [inline src/scss/components/_DefaultForm.scss]: needs improvement

File: src/scss/components/_FancyGrid.scss [hunk 7]
```diff
@@ -76,7 +76,7 @@ $spacing: '(100% - #{$tileSize} * 3) / 2';
             height: 40%;
         }
         h3 {
-            padding: 0 60px;
+            padding: 0 15px;
```
Comment (chapran-leobit-acc) [inline src/scss/components/_FancyGrid.scss]: 0 45px

File: src/scss/components/_ModalWindow.scss [hunk 8]
```diff
@@ -19,9 +19,23 @@
     justify-content: center;
     background-color: $app_bg;
     z-index: 5000;
-    .modal_window_header {
+    overflow-y: auto;
+    @media (max-width:768px) {
```
Comment (chapran-leobit-acc) [inline src/scss/components/_ModalWindow.scss]: needs to have scroll on all of the dimensions

File: src/js/components/lib/ModalWindow.jsx [hunk 9]
```diff
@@ -4,39 +4,41 @@ import ReactDOM from 'react-dom';
...
+        this.el = document.createElement('div');
+        this.closeModal = this.closeModal.bind(this);
+    }
+    closeModal(e) {
+        console.log(e.target);
```
Comment (MatskoVolodia) [inline src/js/components/lib/ModalWindow.jsx]: seriously, guys? :D

**Pull request merged**